packages/solana/test/rpc_client_test.dart

```diff
@@ -9,6 +11,17 @@ import 'config.dart';
 const int _transferredAmount = 0x1000;
 
 void main() {
+  test('throws exception on timeout', () async {
+    final client = RpcClient(
+      devnetRpcUrl,
+      timeout: const Duration(milliseconds: 1),
```

Comment:
Can you make it 0? Because I think we must make sure the test pass, and 1 ms despite being very little it is not zero.

Reply:
Yeah, updated to `0`. Although, I would really like to see this test fail 😄

Reply:
1ms is a very short time indeed :laughing: